packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -332,4 +332,46 @@ describe('<TablePagination />', () => {
       );
     });
   });
+  describe('v5 deprecations', () => {
```

Comment:
Not sure we need to invest time in adding test cases for the deprecations.

Reply:
Removed